website/docs/podman/rosetta.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 20
+title: Native Apple Rosetta translation layer (macOS)
+description: Use Apple Rosetta to speed up cross-architecture containers
+---
+
+# Native Apple Rosetta translation layer
+
+Virtual machine's created by Podman Machine use the native Apple hypervisor `applehv` with Rosetta enabled by default. This increases the speed of any `x86_64` builds or containers to near-native levels by using a translation layer.
+
+For more information, refer to the [official Apple documentation](https://developer.apple.com/documentation/virtualization/running_intel_binaries_in_linux_vms_with_rosetta) regarding the Rosetta technology.
+
+Rosetta support is **enabled by default** on all new Podman Machine installations. If you disable Rosetta, [qemu](https://www.qemu.org/) will instead be used.
```

Comment:
```suggestion
Rosetta support is enabled by default on all new Podman machine installations. If you disable Rosetta, [qemu](https://www.qemu.org/) will instead be used.
```
We should reserve the bold formatting only for actual clickable items in the UI. Otherwise, this formatting might confuse readers.
If we want to highlight this piece of information, we can put it in a note.